client/src/pages/Login.jsx

```diff
@@ -37,13 +31,15 @@ const Login = () => {
         <Typography variant="h4">
           We make it easy for everyone to sell their stuff.
         </Typography>
-        <Button
-          variant="outlined"
-          onClick={googleLoginQuery}
-          startIcon={<GoogleIcon />}
-        >
-          Sign in with Google
-        </Button>
+        <a href="http://localhost:3001/auth/google">
+          <Button
```

Comment:
Will it affect the functionality to use the noreferrer in the a href? <a href="https://..." rel="noreferrer">.

Reply:
Seems to work fine with noreferrer added in the a tag.